### DISCUSSION TITLE
EdgeDB vs postgres-style inheritance issues

### DISCUSSION THREAD

**Discussion by hjalmarlucius** [geldata/gel#3290] [Q&A] [answered]
Does EdgeDB inherit the same issues as postgres wrt inheritance not carrying over uniqueness constraints as per [postgres docs 5.10.1](https://www.postgresql.org/docs/current/ddl-inherit.html#DDL-INHERIT-CAVEATS)? (inheritance pun accidental)

**Comment by msullivan** [comment, marked as answer]
No, exclusive constraints in edgedb are inherited by children and by default apply to both objects together. If you specify the constraint as being `delegated`, then the exclusivity is enforced separately for each child.

**Reply by hjalmarlucius** [reply]
very good news, many thanks!
